rts/Sim/MoveTypes/Systems/GroundMoveSystem.cpp

```diff
@@ -15,9 +17,75 @@
 
 using namespace MoveTypes;
 
-void GroundMoveSystem::Init() {}
+void GroundMoveSystem::Init() {
+    auto& comp = Sim::systemGlobals.CreateSystemComponent<GroundMoveSystemComponent>();
```

Comment:
does it make sense to create a local reference var that immediately goes out of scope? it seems equivalent to
```suggestion
    Sim::systemGlobals.CreateSystemComponent<GroundMoveSystemComponent>();
```
perhaps it's just leftovers?

Reply:
It isn't needed.